src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -355,14 +356,26 @@ private IEventReferenceExpression CreateBoundEventAccessOperation(BoundEventAcce
 
         private IEventAssignmentExpression CreateBoundEventAssignmentOperatorOperation(BoundEventAssignmentOperator boundEventAssignmentOperator)
         {
-            IEventSymbol @event = boundEventAssignmentOperator.Event;
-            Lazy<IOperation> eventInstance = new Lazy<IOperation>(() => Create(boundEventAssignmentOperator.Event.IsStatic ? null : boundEventAssignmentOperator.ReceiverOpt));
+            SyntaxNode syntax = boundEventAssignmentOperator.Syntax;
+            Lazy<IEventReferenceExpression> eventReference = new Lazy<IEventReferenceExpression>(() =>
```

Comment:
Can you refactor this into a separate method `CreateBoundEventReferenceExpressionOperation`?